Update setup.asciidoc (#2052)
elastic/apm-agent-nodejs: docs/setup.asciidoc

[[configuring-the-agent]]
=== Configuring the agent

> Use the below methods to configure the Node.js {agent}, listed in order of prescendence (highest to lowest):

Author: LMK if this rephrase makes sense. Tried to match the other articles.

> Use the below methods to configure the Node.js {agent}, listed in order of prescendence (highest to lowest):

Reviewer: ```suggestion
There are multiple ways to configure the Node.js agent. In order of precedence (higher overwrites lower):
```

})
----

> Below is an example of usage configuring the {agent} to only be active in production:

Reviewer: ```suggestion
This example shows how to configure the agent to only be active in production:
```

[[agent-configuration-file]]
==== Agent configuration file

> The Node.js {agent} looks for a file named `elastic-apm-node.js` in the current working directory.
> Specify a custom path for this file using the <<config-file,`configFile`>> config option.

Reviewer: ```suggestion
The Node.js agent looks for a file named `elastic-apm-node.js` in the current working directory.
You can specify a custom path for this file with the <<config-file,`configFile`>> configuration option.
```

TypeScript,
or the `--experimental-modules` flag for Node.js,
all import statements are evaluated prior to calling any functions.

> **In this case, you can't configure the {agent} by passing a configuration object to the `start` function,

Reviewer: ```suggestion
**In this case, you can't configure the agent by passing a configuration object to the `start` function,
```

import apm from 'elastic-apm-node/start'
----

> Now, configure the {agent} using the environment variables associated with each <<configuration,configuration option>>,

Reviewer: ```suggestion
Now, configure the agent using the environment variables associated with each <<configuration,configuration option>>,
```

or use the optional <<agent-configuration-file,agent configuration file>>.

> To explicitly start the {agent}, import `elastic-apm-node` and call the `apm.start()` method with the

Reviewer: ```suggestion
To explicitly start the agent, import `elastic-apm-node` and call the `apm.start()` method with the
```
